CMakeLists.txt

```diff
@@ -1,148 +1,34 @@
-cmake_minimum_required(VERSION 3.18)
+# Provide general project configuration setup
+#
+cmake_minimum_required(VERSION 3.21)
+
+set(STATUS_QT_VERSION 6.3)
+set(STATUS_VERSION 0.6.0)
```

Comment:
this can be like -DVESRION=0.6.0
`set(STATUS_VERSION ${VERSION})`

Reply:
Do you mean to pass it at build/configuration time? Where is `VERSION` coming from?